packages/fluentui/react-northstar/test/specs/commonTests/isConformant.tsx

```diff
@@ -66,22 +75,20 @@ export function isConformant(
   } = options;
 
   const defaultConfig: IsConformantOptions = {
-    customMount: mountWithProvider,
+    renderOptions: { wrapper: EmptyThemeProvider },
```

Comment:
For reference https://testing-library.com/docs/react-testing-library/api#wrapper